README.md

```diff
@@ -47,6 +47,26 @@ You can use Alfred to
 
 `command + d` : when given "don't save, cancel, and save", triggers "don't save"; also short for saving something to the desktop
 
+## Terminal / Bash
+
+`ctrl + a` : jump to the beginning of line
+
+`ctrl + u` : clear line
```

Comment:
This will clear to the beginning of the line, while ctrl + k will clear to the end, could you edit to clarify?